Cron/FlushInvalidatedCacheTypes.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace Pronko\SelectiveCache\Cron;
+
+use Magento\Framework\App\Cache\TypeListInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Pronko\SelectiveCache\Logger\CleanCacheLogger;
+
+/** Class FlushInvalidatedCacheTypes flushes invalidated cache types by cronjob */
+
+class FlushInvalidatedCacheTypes
+{
+
+    /**
+     * @var TypeListInterface
+     */
+    protected $_cacheTypeList;
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /** @var CleanCacheLogger */
+    protected $logger;
+
+    /**
+     * FlushInvalidatedCache constructor.
+     *
+     * @param TypeListInterfaceAlias $cacheTypeList
+     * @param ScopeConfigInterface $scopeConfig
+     * @param CleanCacheLogger $logger
+     */
+
+    public function __construct(
+        TypeListInterface $cacheTypeList,
+        ScopeConfigInterface $scopeConfig,
+        CleanCacheLogger $logger
+    ) {
+        $this->_cacheTypeList = $cacheTypeList;
+        $this->_scopeConfig = $scopeConfig;
+        $this->_logger = $logger;
+    }
+    
+    /**
+     * Method execute flushes invalidated Cache types
+     *
+     * @return void
+     */
+
+    public function execute()
+    {
+        $cronenabled = $this->getConfig('selectivecache/general/enabled');
```

Comment:
Please use `isFlag()` method

Reply:
I have not worked yet with that method - do you have an example?